quiz/backend/config/settings.py

```diff
@@ -132,3 +127,69 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+# Logging
+
+LOGGING_CONFIG = None
+LOGLEVEL = os.environ.get('LOGGING_LVL', 'DEBUG')
+
+if not os.path.exists(os.path.join(BASE_DIR, 'logs')):
```

Comment:
Przyjacielu, to się będzie odpalać za każdym razem jak będzie się uruchomiać django, to operacja na dysku, kosztowna. https://www.theserverside.com/blog/Coffee-Talk-Java-News-Stories-and-Opinions/gitkeep-push-empty-folders-git-commit